tools/gen_allocs.c

```diff
@@ -589,30 +589,97 @@ gen_dealloc1 ( char * dirname )
   FILE * fp ;
   char  fname[NAMELEN] ;
   char * fn = "deallocs.inc" ;
+  // Open array of deallocs_[n].inc
+  int   numFiles = 12;
+  int    idx      = 0;
+  FILE * fpSub; 
+  char * filename_prefix = "deallocs_" ;
 
   if ( dirname == NULL ) return(1) ;
   if ( strlen(dirname) > 0 ) { sprintf(fname,"%s/%s",dirname,fn) ; }
   else                       { sprintf(fname,"%s",fn) ; }
   if ((fp = fopen( fname , "w" )) == NULL ) return(1) ;
   print_warning(fp,fname) ;
-  gen_dealloc2( fp , "grid%", &Domain ) ;
+
+  fprintf(
+          fp,
+          "INTERFACE\n"
+          );
+  if ( dirname == NULL ) return(1) ;
+  for ( ; idx < numFiles; idx++ )
```

Comment:
Would it be better to initialize `idx = 0` here rather than to assume that no preceding code has modified `idx` since it was initialized at declaration?